### PR TITLE
[docs] update some more quick-start links

### PR DESCRIPTION
**Pull request by polarweasel**

### CODE REVIEW

File: docs/content/v2.6/manage/backup-restore/back-up-data-ycql.md [hunk 1]
```diff
@@ -102,7 +102,7 @@ There are a number of useful options in the `COPY TO` command used to perform th
 
 We are going to use the example shown in the [quick start](../../../quick-start/test-cassandra/) section in order to demonstrate how to create backups.
 
-This section assumes you already have a YugabyteDB cluster. You can install a local cluster on your laptop using [these quick start instructions](../../../quick-start/install/).
+This section assumes you already have a YugabyteDB cluster. You can install a local cluster on your laptop using [these quick start instructions](../../../quick-start/#install-yugabytedb).
```
Comment (ddhodge) [inline docs/content/v2.6/manage/backup-restore/back-up-data-ycql.md:105]: redirects

File: docs/content/v2.6/quick-start/build-apps/go/ysql-pq.md [hunk 2]
```diff
@@ -41,7 +41,7 @@ This tutorial assumes that you have satisfied the following prerequisites.
 
 ### YugabyteDB
 
-YugabyteDB is up and running. If not, please follow these steps in the [Quick Start guide](../../../../quick-start/install/macos/).
+YugabyteDB is up and running. If not, please follow these steps in the [Quick Start guide](../../../../quick-start/#install-yugabytedb).
```
Comment (ddhodge) [inline docs/content/v2.6/quick-start/build-apps/go/ysql-pq.md:44]: can't reach page

File: docs/content/v2.6/quick-start/build-apps/nodejs/ysql-sequelize.md [hunk 3]
```diff
@@ -38,7 +38,7 @@ type: docs
 
 This tutorial assumes that you have:
 
-- installed YugabyteDB and created a universe with YSQL enabled. If not, please follow these steps in the [Quick Start guide](../../../../quick-start/install/macos/).
+- installed YugabyteDB and created a universe with YSQL enabled. If not, please follow these steps in the [Quick Start guide](../../../../quick-start/#install-yugabytedb).
```
Comment (ddhodge) [inline docs/content/v2.6/quick-start/build-apps/nodejs/ysql-sequelize.md:41]: Can't reach page

File: docs/content/v2.6/quick-start/build-apps/ruby/ysql-pg.md [hunk 4]
```diff
@@ -47,7 +47,7 @@ $ gem install pg -- --with-pg-config=<yugabyte-install-dir>/postgres/bin/pg_conf
 
 This tutorial assumes that you have:
 
-- installed YugabyteDB and created a universe with YSQL enabled. If not, please follow these steps in the [Quick Start guide](../../../../quick-start/install/macos/).
+- installed YugabyteDB and created a universe with YSQL enabled. If not, please follow these steps in the [Quick Start guide](../../../../quick-start/#install-yugabytedb).
```
Comment (ddhodge) [inline docs/content/v2.6/quick-start/build-apps/ruby/ysql-pg.md:50]: can't reach page

File: docs/content/v2.6/quick-start/build-apps/ruby/ysql-rails-activerecord.md [hunk 5]
```diff
@@ -37,7 +37,7 @@ type: docs
 
 This tutorial assumes that you have:
 
-- installed YugabyteDB and created a universe with YSQL enabled. If not, please follow these steps in the [Quick Start guide](../../../../quick-start/install/macos/).
+- installed YugabyteDB and created a universe with YSQL enabled. If not, please follow these steps in the [Quick Start guide](../../../../quick-start/#install-yugabytedb).
```
Comment (ddhodge) [inline docs/content/v2.6/quick-start/build-apps/ruby/ysql-rails-activerecord.md:40]: can't reach page

File: docs/content/v2.6/reference/connectors/kafka-connect-yugabytedb.md [hunk 6]
```diff
@@ -79,7 +79,7 @@ For building and using this project, the following tools must be installed on yo
 
 1. Install YugabyteDB and create the database table.
 
-    [Install YugabyteDB and start a local cluster](../../../quick-start/install/).
+    [Install YugabyteDB and start a local cluster](../../../quick-start/#install-yugabytedb).
```
Comment (ddhodge) [inline docs/content/v2.6/reference/connectors/kafka-connect-yugabytedb.md:82]: redirects to preview

File: docs/content/v2.8/explore/fault-tolerance/kubernetes.md [hunk 7]
```diff
@@ -45,7 +45,7 @@ type: docs
 
 YugabyteDB can automatically handle failures and therefore provides [high availability](../../../architecture/core-functions/high-availability/). You will create YSQL tables with a replication factor (RF) of `3` that allows a [fault tolerance](../../../architecture/docdb-replication/replication/) of `1`. This means the cluster will remain available for both reads and writes even if one node fails. However, if another node fails, bringing the number of failures to two, then writes will become unavailable on the cluster in order to preserve data consistency.
 
-If you haven't installed YugabyteDB yet, you can create a local YugabyteDB cluster within five minutes by following the [Quick Start](../../../quick-start/install/) guide.
+If you haven't installed YugabyteDB yet, you can create a local YugabyteDB cluster within five minutes by following the [Quick Start](../../../quick-start/#install-yugabytedb) guide.
```
Comment (ddhodge) [inline docs/content/v2.8/explore/fault-tolerance/kubernetes.md:48]: redirects to preview

File: docs/content/v2.8/deploy/manual-deployment/install-software.md [hunk 8]
```diff
@@ -15,7 +15,7 @@ Install YugabyteDB on each of the nodes using the steps shown below.
 
 ## Download
 
-Download the YugabyteDB binary package as described in the [Quick Start section](../../../quick-start/install/).
+Download the YugabyteDB binary package as described in the [Quick Start section](../../../quick-start/#install-yugabytedb).
```
Comment (ddhodge) [inline docs/content/v2.8/deploy/manual-deployment/install-software.md:18]: redirects to preview

File: docs/content/v2.8/explore/multi-region-deployments/read-replicas-ycql.md [hunk 9]
```diff
@@ -41,7 +41,7 @@ Stale reads are possible with an upper bound on the amount of staleness. Reads a
 
 ## Prerequisites
 
-Ensure that you have downloaded and configured YugabyteDB, as described in [Quick Start](/preview/quick-start/install/macos/).
+Ensure that you have downloaded and configured YugabyteDB, as described in [Quick Start](../../../quick-start/#install-yugabytedb).
```
Comment (ddhodge) [inline docs/content/v2.8/explore/multi-region-deployments/read-replicas-ycql.md:44]: redirects to preview

File: docs/content/v2.8/explore/observability/prometheus-integration/kubernetes.md [hunk 10]
```diff
@@ -47,7 +47,7 @@ You can monitor your local YugabyteDB cluster with a local instance of [Promethe
 
 For details on the metrics targets for YugabyteDB, see [Monitoring with Prometheus](../../../reference/configuration/default-ports/#monitoring-with-prometheus).
 
-If you haven't installed YugabyteDB yet, do so first by following the [Quick Start](../../../quick-start/install/) guide.
+If you haven't installed YugabyteDB yet, do so first by following the [Quick Start](../../../quick-start/#install-yugabytedb) guide.
```
Comment (ddhodge) [inline docs/content/v2.8/explore/observability/prometheus-integration/kubernetes.md:50]: link broken

File: docs/content/v2.8/manage/backup-restore/back-up-data-ycql.md [hunk 11]
```diff
@@ -102,7 +102,7 @@ There are a number of useful options in the `COPY TO` command used to perform th
 
 We are going to use the example shown in the [quick start](../../../quick-start/test-cassandra/) section in order to demonstrate how to create backups.
 
-This section assumes you already have a YugabyteDB cluster. You can install a local cluster on your laptop using [these quick start instructions](../../../quick-start/install/).
+This section assumes you already have a YugabyteDB cluster. You can install a local cluster on your laptop using [these quick start instructions](../../../quick-start/#install-yugabytedb).
```
Comment (ddhodge) [inline docs/content/v2.8/manage/backup-restore/back-up-data-ycql.md:105]: redirects to preview

File: docs/content/v2.8/reference/connectors/kafka-connect-yugabytedb.md [hunk 12]
```diff
@@ -79,7 +79,7 @@ For building and using this project, the following tools must be installed on yo
 
 1. Install YugabyteDB and create the database table.
 
-    [Install YugabyteDB and start a local cluster](../../../quick-start/install/).
+    [Install YugabyteDB and start a local cluster](../../../quick-start/#install-yugabytedb).
```
Comment (ddhodge) [inline docs/content/v2.8/reference/connectors/kafka-connect-yugabytedb.md:82]: redirects

**Review by ddhodge — COMMENTED** [review]
misc comments

File: docs/netlify.toml [hunk 13]
```diff
@@ -242,6 +235,21 @@
...
+[[redirects]]
+  from = "/preview/quick-start/*"
+  to = "/preview/quick-start/:splat"
+  status = 200
+
+# Redirect all other quick-starts to preview
+
 [[redirects]]
   from = ":version/quick-start/*"
   to = "/preview/quick-start-yugabytedb-managed/"
```
Comment (aishwarya24) [inline docs/netlify.toml:255]: @polarweasel - Could you include this redirect under 

# Build an app was in quick-start, now in develop
```
[[redirects]]
  from="/preview/yugabyte-cloud/cloud-quickstart/cloud-build-apps/cloud-add-ip/"
  to = "/preview/develop/build-apps/cloud-add-ip/"
  force = true
```
This one is for the "Before you begin" page under Develop.

**Pull request closed without merging**